config/01-common-auth-defaults.conf

```diff
@@ -13,18 +13,7 @@
 DENY_CLIENT = anonymous@*, unmapped@*
 
 # Authentication settings
-SEC_DEFAULT_AUTHENTICATION = REQUIRED
-SEC_READ_AUTHENTICATION = OPTIONAL
-SEC_CLIENT_AUTHENTICATION = OPTIONAL
-SEC_CLIENT_AUTHENTICATION_METHODS = FS, TOKEN, SCITOKENS, GSI, SSL
-
-# Privacy settings
-SEC_DEFAULT_ENCRYPTION = OPTIONAL
-SEC_DEFAULT_INTEGRITY = REQUIRED
-SEC_READ_INTEGRITY = OPTIONAL
-SEC_CLIENT_INTEGRITY = OPTIONAL
-SEC_READ_ENCRYPTION = OPTIONAL
-SEC_CLIENT_ENCRYPTION = OPTIONAL
+use security:recommended_v9_0
```

Comment:
```suggestion
SEC_CLIENT_AUTHENTICATION_METHODS = FS, TOKEN, SCITOKENS, GSI, SSL
use security:recommended_v9_0
```

Reply:
The default auth method list (as in when neither CLIENT nor DEFAULT param is set) is the same as this explicit value (except it includes CLAIMTOBE, which the server can reject).

Reply:
Gotcha, that makes sense!